docs/intelmqctl.md

```diff
@@ -403,6 +403,13 @@ Use the `-q` or `--quiet` flag to only show non-empty queues:
 intelmqctl: file-output-queue - 234
 ```
 
+The `--sum` or `--count` flag will show the sum of events on all queues:
+```bash
+> intelmqctl list queues --sum
+intelmqctl: 42
```

Comment:
The output I get (and what I expect) is just the number without `intelmqctl: ` prefix. We had that prefix before 5f8feb68a3d1c9faa35e1ad0638380a314b271ec